databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -1830,6 +1909,51 @@ mod tests {
         }
     }
 
+    async fn publish_value(
```

Comment:
why is this part of the PR? Isn't it there yet? Has it changed?

Reply:
That was test specific, I placed it outside to be used by `test_subscribe_case` and `test_subscribe_case_by_id`